src/r_data.c

```diff
@@ -848,6 +848,7 @@ void R_InitColormaps(void)
 
   // [FG] dark/shaded color translation table
   cr_dark = (char *)&colormaps[0][256*15];
+  cr_shaded = (char *)&colormaps[0][256*4];
```

Comment:
Excuse me again, but this is really just a tad darker than normal. So little, that it just feels odd somehow, but it's not clear that the help text is shaded on purpose. It looks like some accidental darkening, like a rendering glitch.

Could we go with e.g. `colormaps[0][256*8];`